content/chat.adoc

```diff
@@ -3,33 +3,19 @@ layout: simplepage
 title: Chat
 ---
 
-= Gitter
+The Jenkins project meets to discuss various topics on Gitter and IRC.
+In addition to the general purpose rooms and channels listed below, link:../sigs/[special interest groups] have their own spaces, and you will see chat rooms linked from the SIGs overview page.
```

Comment:
```suggestion
In addition to the general purpose rooms and channels listed below, link:../sigs/[special interest groups] and link:/projects[sub-projects] have their own spaces, and you will see chat rooms linked from their pages.
```

Reply:
https://github.com/jenkins-infra/jenkins.io/pull/2083 from @sp1r might be a good addition to it

Reply:
Looks like others cannot accept suggestions. So maybe once @daniel-beck accepts this, he can merge?